frontend/src/metabase/admin/settings/selectors.js

```diff
@@ -646,7 +641,7 @@ export const getSections = createSelector(
 
       const settings = section.settings.map(function (setting) {
         const apiSetting =
-          settingsByKey[setting.key] && settingsByKey[setting.key][0];
+          settingsByKey[setting.key] && settingsByKey[setting.key];
```

Comment:
We can remove the element extraction because we change from using `_.groupBy` to `_.indexBy`